TCAT/Cells/LargeDetailTableViewCell.swift

```diff
@@ -15,214 +15,188 @@ protocol LargeDetailTableViewDelegate: class {
+    private let detailLabel = UILabel()
     private var iconView: DetailIconView!
-    private var titleLabel: UILabel!
-
-    private let cellWidth: CGFloat = RouteDetailCellSize.regularWidth
-    private let edgeSpacing: CGFloat = 16
-    private let labelSpacing: CGFloat = 4
-    private let paragraphStyle = NSMutableParagraphStyle()
-    private var cellHeight: CGFloat = RouteDetailCellSize.largeHeight
-    private var direction: Direction!
+    private let titleLabel = UILabel()
+    private let hairline = UIView()
 
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
         setupChevron()
         setupTitleLabel()
         setupDetailLabel()
-    }
 
-    func getTitleLabel() -> UILabel {
-        let titleLabel = UILabel()
-        titleLabel.frame = CGRect(x: cellWidth, y: 0, width: chevron.frame.minX - cellWidth, height: 20)
-        titleLabel.font = .getFont(.regular, size: 14)
-        titleLabel.lineBreakMode = .byWordWrapping
-        titleLabel.textColor = Colors.primaryText
-        titleLabel.text = direction != nil && direction.type == .transfer ? "Bus becomes" : "Board"
-        titleLabel.sizeToFit()
-        return titleLabel
+        setupConstraints()
     }
 
-    func getDetailLabel() -> UILabel {
-        let detailLabel = UILabel()
-        detailLabel.frame = CGRect(x: cellWidth, y: 0, width: 20, height: 20)
-        detailLabel.font = .getFont(.regular, size: 14)
-        detailLabel.textColor = Colors.metadataIcon
-        detailLabel.text = "Detail Label"
-        detailLabel.lineBreakMode = .byWordWrapping
-        detailLabel.sizeToFit()
-        return detailLabel
-    }
-
-    func setupChevron() {
-        chevron = IncreasedTapSizeButton(frame: .zero, sizeIncrease: .init(width: 30, height: 30))
-        chevron.frame.size = CGSize(width: 13.5, height: 8)
-        chevron.frame.origin = CGPoint(x: UIScreen.main.bounds.width - 20 - chevron.frame.width, y: 0)
+    private func setupChevron() {
+        chevron = LargeTapTargetButton(extendBy: 15)
         chevron.setImage(UIImage(named: "arrow"), for: .normal)
         chevron.tintColor = Colors.metadataIcon
         chevron.addTarget(self, action: #selector(chevronButtonPressed), for: .touchUpInside)
         contentView.addSubview(chevron)
     }
 
-    func setupTitleLabel() {
-        titleLabel = getTitleLabel()
+    private func setupTitleLabel() {
+        titleLabel.font = .getFont(.regular, size: 14)
+        titleLabel.lineBreakMode = .byWordWrapping
+        titleLabel.textColor = Colors.primaryText
         contentView.addSubview(titleLabel)
     }
 
-    func setupDetailLabel() {
-        detailLabel = getDetailLabel()
+    private func setupDetailLabel() {
+        detailLabel.font = .getFont(.regular, size: 14)
+        detailLabel.textColor = Colors.metadataIcon
+        detailLabel.lineBreakMode = .byWordWrapping
```

Comment:
nit: you have `lineBreakMode` here but for `titleLabel` you have it after `textColor`